### PR TITLE
Remove warnings

### PR DESCRIPTION
**Pull request by sophiakaz**

### CODE REVIEW

**Review by lucyajiang — COMMENTED** [review]
Question: Why are the Podfile and Manifest changed and what does this mean? 😮

**Comment by jordanF0ster** [comment]
> Question: Why are the Podfile and Manifest changed and what does this mean? 😮

Just an update to cocoa pods

**Review by lucyajiang — APPROVED** [review]
Yay for removing warnings! Never seen int in Obj C (only NSInteger) but if it works it works

File: fbu-app/ViewControllers/HomeScreens/AddHousemateViewController.m [hunk 1]
```diff
@@ -19,6 +19,9 @@ - (IBAction)tapAddHousemates:(id)sender;
 @property (nonatomic, strong) NSMutableArray *potentialHousemates;
 @property (nonatomic, strong) NSMutableArray *housematesToAdd;
 @property (weak, nonatomic) IBOutlet UIButton *houseButton;
+@property (strong, nonatomic) Persona *currentPersona;
```
Comment (jordanF0ster) [inline fbu-app/ViewControllers/HomeScreens/AddHousemateViewController.m:22]: if properties are being used locally only they can be made into instance variables instead
Ex:

    @interface Class ()
    {
        NSString string;
        NSUInteger number;
    }

and then you dont have to access them by using self

**Review by jordanF0ster — APPROVED** [review]

**Pull request merged**